src/SystemTemplates.cc

```diff
@@ -533,6 +604,54 @@ static const unordered_map<uint32_t, ResourceFile::TemplateEntryList> system_tem
     }),
     t_data_eof_hex("Data"),
   }},
+  {RESOURCE_TYPE_itl0, {
+    t_char("Decimal point separator"),
+    t_char("Thousands separator"),
+    t_char("List separator"),
+    t_string("Currency symbol", 3),
+    t_bitfield({
+      t_bool("Leading unit zero"),
+      t_bool("Trailing unit zero"),
+      // 0 = parenthesis, 1 = minus sign
+      t_bool("Negative representation"),
+      t_bool("Currency symbol leads number"),
+    }),

```

Comment:
If a t_bitfield has fewer than 8 t_bools in it, then only the meanings of the highest-order bits are specified (and the rest are ignored). Just want to make sure that's the intended behavior here, and other places in the diff where this may be relevant.

Reply:
That's intended. Originally there were four additional `t_bool("unused")`, but testing showed they aren't necessary, so I removed them to avoid clutter in the output.